addons/account/views/account_move_views.xml

```diff
@@ -1306,6 +1306,8 @@
                     <filter name="open" string="Unpaid" domain="[('state', '=', 'posted'), ('payment_state', 'in', ('not_paid', 'partial'))]"/>
                     <!-- in_payment & paid -->
                     <filter name="closed" string="Paid" domain="[('state', '=', 'posted'), ('payment_state', 'in', ('in_payment', 'paid'))]"/>
+                    <!-- reversed -->
+                    <filter name="reversed" string="Reversed" domain="[('state', '=', 'posted'), ('payment_state', 'in', ('reversed'))]"/>
```

Comment:
Why do we use `in` instead of `=` here? We'll want to add a new state in the domain later? :thinking:

Reply:
@FlorianGilbert Hi! yes, in this way is taking into account the possibility that perhaps in the future it will be necessary to add a new state. 
cc @dbkosky 
Best regards!

Reply:
@FlorianGilbert My bad, it was directly from my suggestion.
@rov-adhoc You can put this instead then, since I think there is no other payment state that the "Reversed" filter would ever include. 
 ```suggestion
                     <filter name="reversed" string="Reversed" domain="[('state', '=', 'posted'), ('payment_state', '=', 'reversed')]"/>
```

Reply:
No problem, keep it like this, I forgot to read the whole conversation.